LabExT/Experiments/StandardExperiment.py

```diff
@@ -460,6 +502,12 @@ def load_measurement_dataset(self, meas_dict, file_path, force_gui_update=True):
         if meas_hash in self.measurements_hashes:
             raise ValueError("Duplicate measurement found!")
 
+        if "sweep_information" not in meas_dict.keys():
+            meas_dict["sweep_information"] = {"part_of_sweep": False, "sweep_association": dict()}
+
+        if "measurement_params" not in meas_dict.keys():
+            meas_dict["measurement_params"] = self.__guess_params(meas_dict)
```

Comment:
same as above, please continue using the existing "measurement settings" key.

Reply:
Done in 73b76d749fac742ecbb6c31cd6045c68d7107887.